peachjam/urls.py

```diff
@@ -286,9 +287,18 @@
         name="document_problem",
     ),
     # Saved Documents
-    path("save-document/", SaveDocumentView.as_view(), name="save_document"),
     path(
-        "save-document/<int:pk>/delete/",
+        "save-document/<int:doc_id>/new",
+        SaveDocumentView.as_view(),
+        name="save_document",
+    ),
+    path(
+        "save-document/<int:doc_id>/update/<int:pk>",
```

Comment:
@actlikewill I'm not sure it makes sense to include the document ID as part of the URL. It's not needed to load the the SavedDocument id.

Reply:
no its not needed to load in the update view, but i added it in the create view, instead of having it in the form. so I left it there to keep the urls consistent